src/conf_mode/nat66.py

```diff
@@ -0,0 +1,176 @@
+
+        nat['deleted'] = ''
+
+        return nat
+
+    # check if NAT66 connection tracking helpers need to be set up - this has to
+    # be done only once
+    if not get_handler(condensed_json, 'PREROUTING', 'NAT_CONNTRACK'):
+        nat['helper_functions'] = 'add'
+
+        # Retrieve current table handler positions
+        nat['pre_ct_conntrack'] = get_handler(condensed_json, 'PREROUTING', 'VYATTA_CT_PREROUTING_HOOK')
+        nat['out_ct_conntrack'] = get_handler(condensed_json, 'OUTPUT','VYATTA_CT_OUTPUT_HOOK')
+    else:
+        nat['helper_functions'] = 'has'
+
+    return nat
+
+def verify(nat):
+    if not nat or 'deleted' in nat:
+        # no need to verify the CLI as NAT66 is going to be deactivated
+        return None
+
+    if 'helper_functions' in nat and nat['helper_functions'] != 'has':
+        if not (nat['pre_ct_conntrack'] or nat['out_ct_conntrack']):
+            raise Exception('could not determine nftable ruleset handlers')
+    
+    if dict_search('source.rule', nat):
+        for rule, config in dict_search('source.rule', nat).items():
+            err_msg = f'Source NAT66 configuration error in rule {rule}:'
+            if 'outbound_interface' not in config:
+                raise ConfigError(f'{err_msg}\n' \
+                                  'outbound-interface not specified')
+            else:
+                if config['outbound_interface'] not in interfaces():
+                    print(f'WARNING: rule "{rule}" interface "{config["outbound_interface"]}" does not exist on this system')
+
+
+            prefix = dict_search('translation.prefix', config)
+            if prefix != None:
+                if not is_ipv6(prefix):
+                    raise ConfigError(f'Warning: IPv6 prefix {prefix} is not a valid address prefix')
+            
+            prefix = dict_search('source.prefix', config)
+            if prefix != None:
+                if not is_ipv6(prefix):
+                    raise ConfigError(f'{err_msg} source-prefix not specified')
+
+
+    if dict_search('destination.rule', nat):
+        for rule, config in dict_search('destination.rule', nat).items():
+            err_msg = f'Destination NAT66 configuration error in rule {rule}:'
+
+            if 'inbound_interface' not in config:
+                raise ConfigError(f'{err_msg}\n' \
+                                  'inbound-interface not specified')
+            else:
+                if config['inbound_interface'] not in 'any' and config['inbound_interface'] not in interfaces():
+                    print(f'WARNING: rule "{rule}" interface "{config["inbound_interface"]}" does not exist on this system')
+
+    return None
+
+def generate(nat):
+    render(iptables_nat_config, 'firewall/nftables-nat66.tmpl', nat, permission=0o755)
+    render(ndppd_config, 'proxy-ndp/ndppd.conf.tmpl', nat, permission=0o755)
+    return None
+
+def apply(nat):
+    if not nat:
+        return None
+    else:
```

Comment:
As you `return` in line 155 there is no need for an else path, please drop the else which saves one level of indention.

Reply:
done